access/slack/README.md

```diff
@@ -1,19 +1,163 @@
 
-Run `make access-slack` from the repository root to build the slack plugin. Then
-you can find it in `./access/slack/build/teleport-slack`.
+## Install the plugin
 
-Detailed install steps are provided in our [docs](https://goteleport.com/docs/enterprise/workflow/ssh-approval-slack/).
+There are several methods to installing and using the Teleport Slack Plugin:
+
+1. Use a [precompiled binary](#precompiled-binary)
+
+2. Use a [docker image](#docker-image)
+
+3. Install from [source](#building-from-source)
+
+### Precompiled Binary
+
+Get the plugin distribution.
+
+```bash
+$ curl -L https://get.gravitational.com/teleport-access-slack-v7.0.2-linux-amd64-bin.tar.gz
+$ tar -xzf teleport-access-slack-v7.0.2-linux-amd64-bin.tar.gz
+$ cd teleport-access-slack
+$ ./install
+```
+
+### Docker Image
+```bash
+$ docker pull quay.io/gravitational/access-plugin-slack:9.0.2
+```
+
+```bash
+$ docker run quay.io/gravitational/access-plugin-slack:9.0.2 version
+teleport-slack v9.0.2 git:teleport-slack-v9.0.2-0-g9e149895 go1.17.8
+```
+
+For a list of available tags, visit [https://quay.io/](https://quay.io/repository/gravitational/access-plugin-slack?tab=tags)
+
+### Building from source
+
+To build the plugin from source you need Go >= 1.16 and `make`.
+
+```bash
+$ git clone https://github.com/gravitational/teleport-plugins.git
+$ cd teleport-plugins/access/slack
+$ make
+$ ./build/teleport-slack start
+```
+
+
+
+## Teleport User and Role
+
+Using Web UI or `tctl` CLI utility, create the role `access-slack` and the user `access-slack` belonging to the role `access-slack`. You may use the following YAML declarations.
+
+### Role
+
+```yaml
+kind: role
+metadata:
+  name: access-slack
+spec:
+  allow:
+    rules:
+      - resources: ['access_request']
+        verbs: ['list', 'read', 'update']
+version: v4
+```
+
+### User
+
+```yaml
+kind: user
+metadata:
+  name: access-slack
+spec:
+  roles: ['access-slack']
+version: v2
+```
+
+## Generate the certificate
+
+For the plugin to connect to Auth Server, it needs an identity file containing TLS/SSH certificates. This can be obtained with tctl:
+
+```bash
+$ tctl auth sign --auth-server=AUTH-SERVER:PORT --format=file --user=access-slack --out=/var/lib/teleport/plugins/slack/auth_id --ttl=8760h
+```
+
+Here, `AUTH-SERVER:PORT` could be `localhost:3025`, `your-in-cluster-auth.example.com:3025`, `your-remote-proxy.example.com:3080` or `your-teleport-cloud.teleport.sh:443`. For non-localhost connections, you might want to pass the `--identity=...` option to authenticate yourself to Auth Server.
+
+## Configuring Slack Plugin
+
+Slack Plugin uses a config file in TOML format. Generate a boilerplate config
+by running the following command:
+
+```
+$ teleport-slack configure > /etc/teleport-slack.yml
+```
+
+Then, edit the config as needed.
+
+```TOML
+# Example slack plugin configuration TOML file
+
+[teleport]
+# This section 
```

Comment:
"This section" seems like a fragment. Admittedly, this may be the default from configure?  If it is verbatim, don't worry about it.  If it isn't verbatim, finish the thought?

Reply:
It was verbatim, however I updated the messaging in the example configuration so that future versions don't include that line when printing the configuration. 

See https://github.com/gravitational/teleport-plugins/pull/473/commits/43d7e12cc82a73db1028d6525c4f8b89137075d2